### PR TITLE
Hero block

### PR DESCRIPTION
**Pull request by ozasadnyy**

### CODE REVIEW

File: scripts/redux/reducer.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+const uiReducer = (state = initialState.ui, action) => {
+  switch (action.type) {
+    case TOGGLE_DRAWER:
+      return Object.assign({}, state, {
+        isDrawerOpened: action.value !== null ? action.value : !state.ui.isDrawerOpened
```
Comment (sophieH29) [inline scripts/redux/reducer.js]: action.value ? action.value : !state.ui.isDrawerOpened

File: src/hoverboard-app.html [hunk 2]
```diff
@@ -90,6 +98,51 @@
         text-transform: uppercase;
       }
 
+      .hero-image {
+        background-color: var(--hero-background-color);
+        transition: background-color 0.3s;
+      }
+
+      app-header iron-pages {
+        z-index: 0;
```
Comment (sophieH29) [inline src/hoverboard-app.html]: why do you need this property?

Comment (ozasadnyy) [inline src/hoverboard-app.html]: yup

File: scripts/redux/reducer.js (hunk 1 above)
Comment (ozasadnyy) [inline scripts/redux/reducer.js]: Or even better `action.value || !state.ui.isDrawerOpened`

**Pull request merged**